fe1-web/src/features/wallet/components/__tests__/RollCallTokensDropDown.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render } from '@testing-library/react-native';
+import React from 'react';
+
+import { mockKeyPair, mockLao, mockRC } from '__tests__/utils';
+
+import { PopToken } from '../../../../core/objects';
+import { RollCallToken } from '../../objects/RollCallToken';
```

Comment:
It might be better to use absolute paths for these given the amount of `../` (especially `PopToken`).